### ISSUE TITLE
Database password rotation problem

### ISSUE THREAD

**Issue by Saba-Shavidze** [terraform-aws-modules/terraform-aws-rds#547]
I have passed `manage_master_user_password_rotation false` but, in the secret manager it rotates the password every 7 days, why?
```ts
variable "manage_master_user_password_rotation" {
  description = " Whether to manage the master user password rotation. Setting this value to false after previously having been set to true will disable automatic rotation"
  type        = bool
  default     = false
}
```

**Comment by ikattur-ota** [comment]
Related to https://github.com/terraform-aws-modules/terraform-aws-rds/issues/538?

**Comment by gpaw789** [comment]
I got caught by this too :(

**Comment by antonbabenko** [comment]
This issue has been resolved in version 6.5.5 :tada:

**Comment by antonbabenko** [comment]
This issue has been resolved in version 6.11.0 :tada:
